tests/languages/docker_test.py

```diff
@@ -1,14 +1,134 @@
+import builtins
+import json
+from typing import List
 from unittest import mock
+from unittest.mock import mock_open
```

Comment:
the `mock` module is already imported -- you can use `mock.mock_open` directly

Reply:
done